integration/test-integration/src/test/resources/apiYaml/api_policies.yaml

```diff
@@ -57,23 +78,11 @@ data:
             parameters:
               headerName: newHeaderKey1
               headerValue: newHeaderVal1
-          -
-            policyName: ccAddHeaderNotSupportedAction
-            policyVersion: v1
-            parameters:
-              headerName: newHeaderKeyFromUnsupportedAction
-              headerValue: newHeaderValueFromUnsupportedAction
```

Comment:
Do we have moved these unsupported action tests to another file?

Reply:
The policy definition has been moved to a different dir `integration/test-integration/src/test/resources/policiesInvalid`. Adding this would mean updating [this](https://github.com/wso2/product-microgateway/blob/cb45c38e649d789bd6f2ac3fedac2d1156847d47/integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/util/ApictlUtils.java#L240) to include the above location. Do you think that it would be better to have an ccAddHeaderNotSupportedAction case?

Reply:
Created an issue to track this. https://github.com/wso2/product-microgateway/issues/3022